flask_peewee/rest/__init__.py

```diff
@@ -848,7 +892,7 @@ def configure_routes(self):
                 self.blueprint.add_url_rule(
                     full_url,
                     '%s_%s' % (api_name, callback.__name__),
-                    self.auth_wrapper(callback, provider),
```

Comment:
Is this possible because of the change to have `authorize` called in every method that is wrapped by `protect`?